.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,46 @@
+name: Doxygen
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  CI_Doxygen:
+    name: ubuntu_debug_x64
+    runs-on: ubuntu-latest
+    container:
+      image: novelrt/novelrt-build:latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Run CI Build Script (Engine & Docs only)
+        run: scripts/cibuild.sh --configuration Debug --generate -DNOVELRT_BUILD_TESTS OFF -DNOVELRT_BUILD_SAMPLES OFF
```

Comment:
IMHO we should make a dedicated parameter for cibuild.sh/build.sh which *only* builds the documentation.